packages/data-model/migrations/0009_add_default_centric_entity.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "dictionary_categories" ALTER COLUMN "active_dictionary_id" SET NOT NULL;--> statement-breakpoint
+ALTER TABLE "dictionary_categories" ADD COLUMN "defaultCentricEntity" varchar;
```

Comment:
```suggestion
ALTER TABLE "dictionary_categories" ADD COLUMN "defaultCentricEntity" varchar;

```

Reply:
This comment will be handled in PR[#105](https://github.com/overture-stack/lyric/pull/105)